components/giftbox.vue

```diff
@@ -0,0 +1,280 @@
+          </a>
+          <!-- eslint-disable-next-line vue/no-v-html -->
+          <div class="feature-content" v-html="selectedFeature.body" />
+        </div>
+        <div v-else class="features-detail">
+          <p>Geen nieuwe features te melden.</p>
+        </div>
+      </div>
+    </b-modal>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'Giftbox',
+    data: () => ({
+      loading: true,
+      features: [],
+      selectedFeature: undefined,
+      seenFeatures: new Set(),
+    }),
+    created() {
+      this.fetchFeatures().then(res => {
+        this.features = res;
+        if (this.features.length > 0) {
+          this.selectedFeature = this.features[0];
+          this.seenFeatures.add(this.selectedFeature.uid);
+        }
+      });  
+    },
+    methods: {
+      async fetchFeatures() {
+        this.loading = true;
+        const response = await fetch(
+          'https://www.publiq.be/uitdatabank/articles.json',
+        );
+        const data = await response.json();
+        this.loading = false;
+        return data.data;
+      },
+      showModal() {
+        if (!this.loading) {
+          this.$refs['giftbox-modal'].show();
+        }
+      },
+      hideModal() {
+        this.$refs['giftbox-modal'].hide();
+      },
+      switchFeature(uid) {
+        if (uid) {
+          this.selectedFeature = this.features.find(
+            (feature) => feature.uid === uid,
+          );
+          this.seenFeatures.add(this.selectedFeature.uid);
+        }
+      },
+    },
+  };
+</script>
+
+<style lang="scss" scoped>
+  .giftbox-wrapper {
+    width: 3rem;
+    height: 3rem;
+    line-height: 3rem;
+    position: absolute;
+    right: 1.5rem;
+    bottom: 1.5rem;
+    text-align: center;
+
+    .giftbox-button {
+      width: 100%;
+      height: 100%;
+      line-height: 100%;
+      border-radius: 50%;
+      background-color: #c0120c;
+      text-align: center;
+      -webkit-box-shadow: 0px 0px 35px -8px rgba(0, 0, 0, 0.75);
+      -moz-box-shadow: 0px 0px 35px -8px rgba(0, 0, 0, 0.75);
+      box-shadow: 0px 0px 35px -8px rgba(0, 0, 0, 0.75);
+      transition: background-color 500ms, transform 500ms;
+
+      &:active {
+        background-color: lighten($color: #c0120c, $amount: 40%);
+        transform: scale(0.75, 0.75);
+      }
+    }
+
+    .giftbox-icon {
+      color: #fff;
+      width: 65%;
+      height: 65%;
+      position: relative;
+      top: 17.5%;
+    }
+  }
+</style>
+
+<style lang="scss">
+  $grey: #efefef;
+
+  .modal-open .modal {
+    overflow-y: hidden;
+    height: 95vh;
+  }
+
+  .modal-body {
+    padding: 0;
+  }
+
+  .modal-content {
+    height: 100vh;
+  }
+
+  .features-wrapper {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    border-top: $grey 2px solid;
+
+    .features-list {
+      width: 30%;
+      height: 87vh;
+      border-right: $grey 2px solid;
+      overflow-y: scroll;
+
+      .feature-title-block {
```

Comment:
Don't nest too many levels
`Be aware that overly nested rules will result in over-qualified CSS that could prove hard to maintain and is generally considered bad practice.`

I don't think it's necessary to nest these classes they are already quite specific.